File: SimpleAccounting/app/src/main/java/com/emmanuelmess/simpleaccounting/activities/SettingsActivity.java
package com.emmanuelmess.simpleaccounting.activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.emmanuelmess.simpleaccounting.R;
import com.emmanuelmess.simpleaccounting.fragments.settings.MainSettingsFragment;

public class SettingsActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);

		setContentView(R.layout.activity_settings);
	}
}

File: SimpleAccounting/app/src/main/java/com/emmanuelmess/simpleaccounting/activities/SettingsActivity.kt
package com.emmanuelmess.simpleaccounting.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.emmanuelmess.simpleaccounting.R
import com.emmanuelmess.simpleaccounting.fragments.settings.MainSettingsFragment

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_settings)
    }
}
